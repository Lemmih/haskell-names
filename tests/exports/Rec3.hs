module Rec3 (rec1_1, rec1_2, rec3) where

import Rec2
import Rec1 (rec1_2)

rec3 = rec3
