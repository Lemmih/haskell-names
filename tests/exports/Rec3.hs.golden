Symbols (
  fromList
    [ SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "Rec1"
                                          "rec1_1"
                          }
        , sv_fixity = Nothing
        }
    , SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "Rec1"
                                          "rec1_2"
                          }
        , sv_fixity = Nothing
        }
    , SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "Rec3"
                                          "rec3"
                          }
        , sv_fixity = Nothing
        }
    ] ) (
  fromList [] )